### PR TITLE
Update Docker documentation

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: docs/docker.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-Our test runners are deployed as docker images.
+Our test runners are deployed as Docker images.
 
-- Your dockerfile should live at the root directory of your repository and it should be called `Dockerfile`
-- Your dockerfile should be the minimal needed for your test runners and create the minimal image needed for running the tests.
-  It should use alpine linux if possible.
-- The working directory should be `/opt/test-runner`
+- Your dockerfile should live at the root directory of your repository and it should be called `Dockerfile`.
+- Your dockerfile should create the minimal image needed for your test runner to run the tests.
+  It should use alpine linux if possible. Applying the official [Dockerfile best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) can help to create a minimal image.
```
Comment (iHiD) [inline docs/docker.md:7]: Do we actually care about alpine linux? Or is it purely that it's small. Should we flip this and say "Applying the official..." first and then "The minimal-sized image is often based on alpine linux"

File: docs/docker.md [hunk 2]
```diff
@@ -15,4 +15,4 @@ More background information and optional hints:
 - To run the containers in production the `runc` command is used which does not use the `ENTRYPOINT` specified in the `Dockerfile`.
   It would still be good to have an `ENTRYPOINT` specified for local use and testing.
 - The students solution is currently mounted to `/mnt/exercism-iteration/` in production.
-  We still pass in the folder as a second argument to the analyzer in case that changes in the future.
+  We still pass in the solution folder as the second argument to the analyzer in case that changes in the future.
```
Comment (iHiD) [inline docs/docker.md:18]: You could change this now too?

**Review by iHiD — APPROVED** [review]

File: docs/docker.md (hunk 2 above)
Comment (ErikSchierboom) [reply docs/docker.md:18]: I'll rephrase a bit

File: docs/docker.md (hunk 1 above)
Comment (ErikSchierboom) [reply docs/docker.md:7]: I think that makes sense. Let me update.

File: docs/docker.md (hunk 2 above)
Comment (ErikSchierboom) [reply docs/docker.md:18]: Do we even need this bit of text?

Comment (ErikSchierboom) [reply docs/docker.md:18]: If we don't mention this, maintainers also can't rely on it.

Comment (iHiD) [reply docs/docker.md:18]: Yeah - get rid of it.

**Review by iHiD — APPROVED** [review]

**Pull request merged**